views/audit/bigquery_audit_log_v2.sql

```diff
@@ -0,0 +1,1161 @@
+        '$.jobChange.job.jobConfig.queryConfig.priority')) AS queryConfigPriority,
+    COALESCE(
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobInsertion.job.jobConfig.queryConfig.destinationTableEncryption.kmsKeyName'),
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobConfig.queryConfig.destinationTableEncryption.kmsKeyName')
+      ) AS queryConfigDestinationTableEncryptionKmsKeyName,
+    COALESCE(
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobInsertion.job.jobConfig.queryConfig.statementType'),
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobConfig.queryConfig.statementType')) AS queryConfigStatementType,
+    SPLIT(
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobConfig.queryConfig.destinationTable'),
+      "/")[SAFE_OFFSET(1)] AS queryConfigDestinationTableProjectId,
+    SPLIT(
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobConfig.queryConfig.destinationTable'),
+      "/")[SAFE_OFFSET(3)] AS queryConfigDestinationTableDatasetId,
+    SPLIT(
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobConfig.queryConfig.destinationTable'),
+      "/")[SAFE_OFFSET(5)] AS queryConfigDestinationTableId,
+    /*
+     * Describes a load job.
+     * https://cloud.google.com/bigquery/docs/reference/auditlogs/rest/Shared.Types/BigQueryAuditMetadata#load
+     */
+    SPLIT(TRIM(TRIM(
+      JSON_EXTRACT(protopayload_auditlog.metadataJson,
+        '$.jobChange.job.jobStats.queryStats.sourceUris'),
+      '["'), '"]'), '","') AS loadConfigSourceUris,
+    CAST(JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.sourceUrisTruncated') AS BOOL) AS loadConfigSourceUrisTruncated,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.schemaJson') AS loadConfigSchemaJson,
+    CAST(JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.schemaJsonUrisTruncated') AS BOOL) AS loadConfigSchemaJsonTruncated,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.destinationTable') AS loadConfigDestinationTable,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.createDisposition') AS loadConfigCreateDisposition,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.writeDisposition') AS loadConfigWriteDisposition,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+      '$.jobChange.job.jobConfig.loadConfig.destinationTableEncryption.kmsKeyName'
+      ) AS loadConfigDestinationTableEncryptionKmsKeyName,
+    /*
+     * Describes an extract job.
```

Comment:
I believe we need COALESCE here for extractConfig, same reason as for loadConfig